audit-worker-versions/main.go

```diff
@@ -24,6 +24,41 @@ type (
 	Queue tcqueue.Queue
 )
 
+type WorkerInfo struct {
+	WorkerPoolID   string
+	Implementation string
+	Version        string
+	Details        map[string]string
+	hasNoArtifacts bool
+	isUnknown      bool
+}
+
+func (w *WorkerInfo) info() string {
```

Comment:
```suggestion
func (w *WorkerInfo) String() string {
```

This would be idiomatic, see [Stringer](https://go.dev/tour/methods/17) interface.